### PR TITLE
feat(chat-e2e): view shared prompt tests

### PR DESCRIPTION
**Pull request by irinakartun**
**Description:**

implemented view shared prompt tests

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [ ] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2059.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat-e2e/src/assertions/downloadAssertion.ts [hunk 1]
```diff
@@ -9,4 +11,14 @@ export class DownloadAssertion {
     expect(downloadedData.path).toBeTruthy();
     expect(downloadedData.path).toMatch(new RegExp(`${expectedExtension}$`));
   }
+
+  public async assertFileIsDownloaded(downloadedData: UploadDownloadData) {
+    const downloadedFiles = FileUtil.getExportedFiles();
+    expect
+      .soft(
+        downloadedFiles?.find((f) => f.includes(downloadedData.path)),
```
Comment (nartovm) [inline apps/chat-e2e/src/assertions/downloadAssertion.ts]: shouldn't we check that the size of the file is > 0?

**Review by nartovm — DISMISSED** [review]

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2059.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat-e2e/src/assertions/downloadAssertion.ts (hunk 1 above)
Comment (irinakartun) [reply apps/chat-e2e/src/assertions/downloadAssertion.ts]: added undefined value check

**Review by nartovm — APPROVED** [review]

**Pull request merged**